Tests.Integration.Presentation.Web/Tools/ItSystemHelper.cs

```diff
@@ -155,6 +156,23 @@ public static async Task<HttpResponseMessage> DeleteItSystemAsync(int systemId,
             return await HttpApi.PatchWithCookieAsync(url, cookie, body);
         }
 
+        public static async Task<HttpResponseMessage> SendSetUuidRequestAsync(
```

Comment:
lad os fjerne denne også.. og så skal vi lige have en opgave i backlog på at blokere for den patch

Reply:
https://os2web.atlassian.net/browse/KITOSUDV-1623